pkg/common/resources/resources.go

```diff
@@ -363,18 +363,20 @@ func Sub(left, right *Resource) *Resource {
 	return out
 }
 
-// SubOnlyExisting subtract delta from defined resource.
-// Ignore any type not defined in the base resource (ie receiver).
-// Used as part of the headroom updates as undefined resources are unlimited
-func (r *Resource) SubOnlyExisting(delta *Resource) {
+// SubOnlyExisting subtracts delta from base resource.
+// Ignores any type not defined in the base resource (ie receiver).
+// Used as part of the headroom updates as undefined resources are unlimited.
```

Comment:
No 'receiver', no 'the headroom updates' here anymore, we could align the description with AddOnlyExisting.

Reply:
Done. The description has been aligned with AddOnlyExisting.